src/app/layout.tsx

```diff
@@ -0,0 +1,10 @@
+import { ReactNode } from "react";
+import "./globals.scss";
+
+type Props = {
+  children: ReactNode;
+};
+
+export default function RootLayout({ children }: Props) {
```

Comment:
we could move `src/app/[locale]/components/Layout/Layout.tsx` here, but keep the Navbar and Footer in the components folder.

Reply:
🤔 hmmm... Okay. Hadn't crossed my mind.

Reply:
I am a little confused by this.

I used this structure to catch errors (create an error boundary) because I was tired of getting those ugly error message windows when I could not get the locale to work. [next-intl docs](https://next-intl-docs.vercel.app/docs/environments/error-files).

Are you suggesting to move the `src/app/[locale]/compnents/Layout/Layout.tsx` file to `src/app` directory?
Would it not be better to place it in `src/app/[locale]/` directory since it is the layout that is used with a locale.

Reply:
I see what you mean now, this layout is not doing much but it seems that is required because the build fails when it's missing.
We can just leave it as it is, only one thing, could you move BaseLayout and its modules to the components folder?